tests/no_version/test_unit_test_zip_folder_to_byte_stream.py

```diff
@@ -1,39 +1,72 @@
+import os
+import shutil
 from pathlib import Path
 from zipfile import ZipFile
 
-from devtools import debug
-
-from fractal_server.app.routes.aux._job import _zip_folder_to_byte_stream
+from fractal_server.zip_tools import _zip_folder_to_byte_stream
+from fractal_server.zip_tools import _zip_folder_to_file_and_remove
 
 
 def test_zip_folder_to_byte_stream(tmp_path: Path):
```

Comment:
This is testing two different functions. Let's split into two different unit tests.